test/teos/unit/test_block_processor.py

```diff
@@ -13,9 +14,9 @@
 )
 
 
-def test_get_best_block_hash(run_bitcoind, block_processor):
+def test_get_best_block_hash(block_processor):
     best_block_hash = block_processor.get_best_block_hash()
-    # As long as bitcoind is running (or mocked in this case) we should always a block hash
+    # As long as bitcoind is running we should always a block hash
```

Comment:
typo "always a block hash"

Reply:
Fixed

Reply:
Still missing the verb in the sentence?

Reply:
I think that's showing the old one:

https://github.com/talaia-labs/python-teos/blob/a79ec735b7c441686eef6070dd9dba6c2e571700/test/teos/unit/test_block_processor.py#L19

Reply:
Oh, weird, I tried opening the file and still saw the old one.